_conferences/mdevcamp-2020.md

```diff
@@ -5,4 +5,7 @@ location: Prague, Czech Republic
 
 date_start: 2020-05-29
 date_end:   2020-05-29
+
+cfp_end:    2020-02-29
```

Comment:
A cfp_start is needed. If the CFP is already open you can just set it to today.

Reply:
OK. Added.